app/controllers/mentoring/automation_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Mentoring::AutomationController < ApplicationController
+  before_action :ensure_mentor!
```

Comment:
We'll want a new method for this. Let's call them `supermentors` (and we can rep-gate and num_solutions and average_rating that)